packages/x-license-pro/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "5.4.0",
+  "version": "5.5.0",
```

Comment:
Something looks broken here. There were no changes in this package https://github.com/mui/mui-x/compare/v5.4.0...v5.5.0. I would expect no diffs

```diff
  "version": "5.4.0",
```